client/src/components/publications/publicationsLayout/PublicationsEditor.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import { Dropdown } from 'react-bootstrap';
+import { useDispatch } from 'react-redux';
+import styled from 'styled-components';
+import { PropTypes } from 'prop-types';
+import { sortingOptions, groupByOptions } from '../../../config/publications';
+import { updatePublicationOptions } from '../../../actions/website';
+import '../publications.css';
+
+export const StyledButtonGroup = styled.div`
+  background-color: transparent;
+  border-radius: 0;
+  padding: 5px;
+  color: black;
+`;
+
+export const ButtonGroupItem = styled.button`
+  background: #ededed;
+  border: 1px solid ${(props) => props.borderColor || '#ccc'};
+  padding: 1px 3px;
+  border-radius: 3px;
+  cursor: pointer;
+  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande";
+  width: auto;
+  transition: all 0.2s ease 0s;
+  color: ${(props) => props.color};
+  &:hover{
+    background: ${(props) => props.hoverBorderColor || 'lightgrey'};
+    color: ${(props) => props.hoverColor};
+  }
+  ${({ press }) => (press
+    && 'color: #4d4d4d; background: #ededed; outline: none; -webkit-box-shadow: inset 0px 0px 10px #c1c1c1; -moz-box-shadow: inset 0px 0px 10px #c1c1c1; box-shadow: inset 0px 0px 10px #c1c1c1;&:hover { background: #BEBEBE }'
+  )
+}
+`;
+
+export const EditorButton = styled.button`
+  color: white;
+  border: 1px solid #ccc;
+  cursor: pointer;
+  width: auto;
+  background-color: #007bff;
+  border-color: #007bff;
+  padding: .25rem .5rem;
+  font-size: .875rem;
+  line-height: 1.5;
+  border-radius: .2rem;
+  &:hover{
+    background-color: #0069d9;
+  }
+`;
+
+const PublicationsEditor = ({
+  options,
+  setOptions,
+  publications,
+  teamId,
+  sortPublications,
+  setShowCreateForm,
+  setShowImportForm,
+}) => {
+  const dispatch = useDispatch();
+
+  const handleUpdate = () => {
+    dispatch(updatePublicationOptions(teamId, options));
+  };
+  return (
+    <>
+      <StyledButtonGroup>
+        <Dropdown>
+          <Dropdown.Toggle size="sm">
+            Add
+          </Dropdown.Toggle>
+          <Dropdown.Menu>
+            <Dropdown.Item onClick={() => setShowCreateForm(true)}>Insert Publication</Dropdown.Item>
+            <Dropdown.Item onClick={() => setShowImportForm(true)}>Import Publications</Dropdown.Item>
+          </Dropdown.Menu>
+        </Dropdown>
+      </StyledButtonGroup>
+      <StyledButtonGroup>
+        Group By
```

Comment:
Maybe have some padding in between these filters, rn it's cluttered , so maybe space it evenly and have the update layout at the edge

Reply:
> ![image](https://user-images.githubusercontent.com/73247453/133792541-68d40283-9422-4790-a484-e614b85c25e9.png)
> 
> Idk if this is a bug from before! But the author field won't let you enter multiple characters at once.

Thanks for catching this, not sure when this bug exists but I will fix this in this PR.